internal/backends/qt/build.rs

```diff
@@ -15,6 +15,7 @@ fn main() {
             "cargo:warning=Could not find a Qt installation. The Qt backend will not be functional. \
             See https://github.com/slint-ui/slint/blob/master/docs/install_qt.md for more info"
         );
+        println!("cargo:warning=    {}", std::env::var("DEP_QT_ERROR_MESSAGE").unwrap());
```

Comment:
Just curious, where are there a leading spaces?

Reply:
So it feels indented compared to the previous message  (to show that they belong together)